engine/classes/Elgg/Di/ServiceProvider.php

```diff
@@ -751,12 +751,19 @@ public function initConfig(Config $config, ServiceProvider $sp) {
 		$lock('dataroot');
 
 		if ($config->cacheroot) {
-			$config->cacheroot = rtrim($config->cacheroot, '\\/') . DIRECTORY_SEPARATOR;
+			$config->cacheroot = Paths::sanitize($config->cacheroot);
 		} else {
-			$config->cacheroot = $config->dataroot . 'caches/';
+			$config->cacheroot = Paths::sanitize($config->dataroot . 'caches');
 		}
 		$lock('cacheroot');
 
+		if ($config->assetroot) {
+			$config->assetroot = Paths::sanitize($config->assetroot);
+		} else {
+			$config->assetroot = Paths::sanitize('views_simplecache');
```

Comment:
shouldn't this be something like `$config->cachroot . 'views_simplecache'`?

Reply:
I mean in `Paths::sanitize`

Reply:
Yes, it should. Question for you, do we want to keep these BC? If someone has a symlink setup for the old location, we would probably want to keep it there, no?

Reply:
Or rewrite the symlink to the new location (if possible). If not possible maybe add admin notice.

I think don't try to hard to keep BC, just mention something should manually be done